bundle.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM scratch
```

Comment:
One thing: although this file is autogenerated, it would be good to have a documentation (or a link) that explains what all the labels means.

Reply:
I have added a link in readme.md file when the `make bundle` step, to the documentation about the labels.

Reply:
- https://github.com/operator-framework/operator-registry/blob/master/docs/design/operator-bundle.md#bundle-annotations
- https://olm.operatorframework.io/docs/tasks/creating-operator-bundle/#contents-of-annotationsyaml-and-the-dockerfile